classes/ProfileManager.php

```diff
@@ -1,4 +1,7 @@
 <?php
+
+use function PHPUnit\Framework\returnValue;
```

Comment:
I don't remember typing this. Perhaps it's something my IDE did automatically when I named a variable $returnValue? Don't know what this is about....